From 6c66cfb0606ed9efe7fc72495a8293aec86e6bf5 Mon Sep 17 00:00:00 2001
From: yuhuawei <yuhuawei.123@bytedance.com>
Date: Sat, 24 May 2025 22:53:54 +0800
Subject: [PATCH] feat: disable volc esp libs

---
 components/clouds/CMakeLists.txt | 9 +++++----
 1 file changed, 5 insertions(+), 4 deletions(-)

diff --git a/components/clouds/CMakeLists.txt b/components/clouds/CMakeLists.txt
index ad734a1c..30efc462 100644
--- a/components/clouds/CMakeLists.txt
+++ b/components/clouds/CMakeLists.txt
@@ -3,7 +3,8 @@ if (DEFINED ENV{DUER_PATH})
     return()
 endif()
 
-set(COMPONENT_ADD_INCLUDEDIRS ./dueros/lightduer/include ./volc_engine_rtc/include)
+# set(COMPONENT_ADD_INCLUDEDIRS ./dueros/lightduer/include ./volc_engine_rtc/include)
+set(COMPONENT_ADD_INCLUDEDIRS ./dueros/lightduer/include)
 
 # Edit following two lines to set component requirements (see docs)
 set(COMPONENT_SRCS)
@@ -14,9 +15,9 @@ target_link_libraries(${COMPONENT_TARGET} INTERFACE "-L${CMAKE_CURRENT_LIST_DIR}
 
 IF ((CONFIG_IDF_TARGET STREQUAL "esp32s3") OR (CONFIG_IDF_TARGET STREQUAL "esp32") OR (CONFIG_IDF_TARGET STREQUAL "esp32c5"))
 IF (IDF_VERSION_MAJOR EQUAL 5)
-add_prebuilt_library(VolcEngineRTCLite "${CMAKE_CURRENT_LIST_DIR}/volc_engine_rtc/libs/${CONFIG_IDF_TARGET}/libVolcEngineRTCLite.a"
-                    REQUIRES mbedtls espressif__zlib json lwip)
-target_link_libraries(${COMPONENT_LIB} INTERFACE VolcEngineRTCLite)
+# add_prebuilt_library(VolcEngineRTCLite "${CMAKE_CURRENT_LIST_DIR}/volc_engine_rtc/libs/${CONFIG_IDF_TARGET}/libVolcEngineRTCLite.a"
+#                     REQUIRES mbedtls espressif__zlib json lwip)
+# target_link_libraries(${COMPONENT_LIB} INTERFACE VolcEngineRTCLite)
 ENDIF()
 ENDIF()
  
-- 
2.44.0

